content/com/github/ldapchai/ldapchai-0.8.6.jar.diffoscope

```diff
@@ -0,0 +1,1068 @@
+--- /reference/ldapchai-0.8.6.jar
++++ /rebuild/ldapchai-0.8.6.jar
+├── com/novell/ldapchai/impl/AbstractChaiEntry.class
+│ ├── procyon -ec {}
+│ │ @@ -51,15 +51,15 @@
+│ │              return true;
+│ │          }
+│ │          if (!(o instanceof ChaiEntry)) {
+│ │              return false;
+│ │          }
+│ │          final AbstractChaiEntry chaiEntry = (AbstractChaiEntry)o;
+│ │          if (this.chaiProvider != null) {
+│ │ -            if (!this.chaiProvider.equals((Object)chaiEntry.chaiProvider)) {
+│ │ +            if (!this.chaiProvider.equals(chaiEntry.chaiProvider)) {
+│ │                  return false;
+│ │              }
+│ │          }
+│ │          else if (chaiEntry.chaiProvider != null) {
+│ │              return false;
+│ │          }
+│ │          if (this.entryDN != null) {
+│ │ @@ -206,16 +206,16 @@
+│ │          int i = 0;
+│ │          while (i < length) {
+│ │              final byte[] value = array[i];
+│ │              final String strValue = new String(value, Charset.forName(characterEncoding));
+│ │              final int sepPos = strValue.indexOf(35);
+│ │              final int typeInt = Integer.parseInt(strValue.substring(0, sepPos));
+│ │              final AbstractChaiEntry.NetworkAddressType type = AbstractChaiEntry.NetworkAddressType.forIdentifier(typeInt);
+│ │ -            switch (type.ordinal()) {
+│ │ -                case 0: {
+│ │ +            switch (AbstractChaiEntry.AbstractChaiEntry$1.$SwitchMap$com$novell$ldapchai$impl$AbstractChaiEntry$NetworkAddressType[type.ordinal()]) {
+│ │ +                case 1: {
```

Comment:
It is quite weird how the switch condition changes.
At bytecode level, the change is:

```diff
-        74: aload_3
-        75: invokevirtual #175                // Method com/novell/ldapchai/cr/HashSaltAnswer$VERSION.ordinal:()I
-        78: lookupswitch  { // 2
-                       0: 104
-                       1: 123
-                 default: 157
+        74: getstatic     #175                // Field com/novell/ldapchai/cr/HashSaltAnswer$1.$SwitchMap$com$novell$ldapchai$cr$HashSaltAnswer$VERSION:[I
+        77: aload_3
+        78: invokevirtual #181                // Method com/novell/ldapchai/cr/HashSaltAnswer$VERSION.ordinal:()I
+        81: iaload
+        82: lookupswitch  { // 2
+                       1: 108
+                       2: 127
+                 default: 161
```

There seems to a field access, `getstatic`, which is quite strange.